test/utils/parent-component.jsx

```diff
@@ -18,12 +34,19 @@ export class ParentComponentWithContext extends Component {
     }
 
     render() {
+        // If we need to use functions on the child element, then add a 'ref' propType
+        const childElement = this.props.accessElement ? React.Children.map(
```

Comment:
`react-redux` uses `withRef` to tell if the the wrapped component should be accessible. I like that better than `accessElement` /shrug